### DISCUSSION TITLE
C'est quoi un Pipe ? | Bonjour Angular

### DISCUSSION THREAD

**Discussion by giscus** [bonjour-angular/apprendre#25] [General]
# C'est quoi un Pipe ? | Bonjour Angular

C'est quoi un Pipe dans Angular ?

https://apprendre.bonjour-angular.com/cest-quoi/pipe/

**Comment by AurelienPREVOST** [comment]
Une idée pour placer le currency en bout de chaine de caractère sans faire de manipulation sur la chaine de caractere en JS?
